pennylane_lightning/src/benchmarks/AllOperations.hpp

```diff
@@ -0,0 +1,214 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+#include "Bench_Utils.hpp"
+
+#include "Constant.hpp"
+#include "StateVectorManaged.hpp"
+
+#include <algorithm>
+#include <random>
+#include <vector>
+
+constexpr size_t num_gates = 32;
+constexpr size_t num_gntrs = 32;
+constexpr size_t num_matrices = 32;
+
```

Comment:
Will these need to be updated for future custom/specific gates that may get added?